pytest/lib/cluster.py

```diff
@@ -508,10 +510,10 @@ def reset_validator_key(self, new_key):
 
 
 def github_auth():
-    print("Go to the following link in your browser:")
-    print()
-    print("http://nayduck.eastus.cloudapp.azure.com:3000/local_auth")
-    print()
+    logger.info("Go to the following link in your browser:")
+    logger.info("")
+    logger.info("http://nayduck.eastus.cloudapp.azure.com:3000/local_auth")
+    logger.info("")
```

Comment:
Like before, this is an interactive function. I don’t think this should be converted to use logger.

Reply:
As mentioned below by @mina86 , we can also drop the `logger.info("")`